src/S3/Internals.elm

```diff
@@ -216,6 +215,15 @@ policyToJson policy =
 
 
 
+-- URL --
+
+
+makeUrl : Config -> String
+makeUrl (Config record) =
+    "https://" ++ record.bucket ++ "." ++ record.awsS3Host
```

Comment:
There's no reason to have the string interpolation dependency. It seems better to put this in the internal files as well.